backend/app/main.py

```diff
@@ -80,7 +81,20 @@ async def check_auth(request: Request, call_next):
     allow_headers=["*"],
 )
 
-# returns a list of {'word': 'word', 'sentence': 'sentence'}
+@app.get('/suggestion')
+async def get_suggestion(request: Request, question: str, notes: str):
+    try:
+        userId = request.state.decoded_token['uid']
+        response = table.get_item(Key={'UserId': userId})
+        resume = response['Item']['Resume']
+    except Exception as e:
+        logging.error("Error occurred while fetching user's resume data")
+        return JSONResponse(content="Error occurred while processing your request", status_code=HTTP_500_INTERNAL_SERVER_ERROR)
+
+    gpt_response = gpt_utils.GPTUtils(config.OPENAI_KEY).get_suggestion(question, notes, resume)
+    return StreamingResponse(gpt_response, media_type="text/event-stream")
+
+# Rest of the code remains the same
```

Comment:
change the comment back to "# returns a list of {'word': 'word', 'sentence': 'sentence'}"